zerver/views/webhooks/gocd.py

```diff
@@ -0,0 +1,41 @@
+
+from __future__ import absolute_import
+from zerver.models import get_client
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+import ujson
+
+@api_key_only_webhook_view
+@has_request_variables
+def api_gocd_webhook(request, user_profile, stream=REQ(default='gocd')):
+    try:
+        payload = ujson.loads(request.body)
+    except ValueError as e:
+        return json_error("Malformed JSON input, ", e.message)
```

Comment:
you can now use `payload=REQ(argument_type="body")` in the arguments rather than doing the JSON parsing manually in this function.